projects/shared/src/lib/pipe/entity-label.pipe.ts

```diff
@@ -0,0 +1,58 @@
+/*
+ * RERO ILS UI
+ * Copyright (C) 2019-2023 RERO
+ * Copyright (C) 2019-2023 UCLouvain
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, version 3 of the License.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+import { Pipe, PipeTransform } from '@angular/core';
+import { TranslateService } from '@ngx-translate/core';
+import { AppSettingsService } from '../service/app-settings.service';
+
+@Pipe({
+  name: 'entityLabel'
+})
+export class EntityLabelPipe implements PipeTransform {
+
+  private _authorizedKey = 'authorized_access_point';
+  private _defaultPartSeparator = ' - '
+
+  /**
+   * Constructor
+   * @param _translateService - TranslateService
+   */
+  constructor(
+    private _translateService: TranslateService
+  ) {
+  }
+
+  /**
+   * Pipe transformation method.
+   *   Get the best possible entity label depending on the current locale language used.
+   *   If entity has some `subdivisions` parts, they will be join using after the main label.
```

Comment:
Extra space

Reply:
it's a conscious choice. I think the docstring is more readable that way : A main docstring line, additional lines give more explainations.